src/components/auth.tsx

```diff
@@ -56,9 +62,16 @@ const Dropdown: React.FC<BoxProps & { onSignOut?: () => void; show?: boolean }>
     );
   }
 );
+
+const AccountNameComponent = memo(() => {
+  const address = useCurrentAddress();
+  const accountName = useAccountName(address);
+  return <Text>{accountName?.names}</Text>;
```

Comment:
This is an empty array atm, but not sure why yet.

Reply:
BNS names are network specific. meaning if you have fara.btc on mainnet, it won't show up on testnet

Reply:
Ok, got it, so prob won't display a BNS name very often so would just show if there is one. I am a little confused about why this is an array if a BNS name is tied to one account?

Reply:
You can have many off chain names, eg fara.stx.id